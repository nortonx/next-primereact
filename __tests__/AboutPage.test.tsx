import { render, screen } from "@testing-library/react"
import AboutPage from "@/app/about/page"
import { describe, it, expect } from "@jest/globals"
describe("About Page", () => {
  it("should render and match snapshot", async () => {
    const { container } = render(<AboutPage/>)
    const page = await screen.findByTestId("about-page")
    expect(page).toBeDefined()
    expect(container).toMatchSnapshot()
  })
})