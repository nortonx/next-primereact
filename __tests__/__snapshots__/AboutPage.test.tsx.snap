// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page should render and match snapshot 1`] = `
<div>
  <main
    class="flex align-items-center justify-content-center"
    data-testid="about-page"
  >
    <h1>
      About Page
    </h1>
  </main>
</div>
`;
